packages/ag-grid-theme/css/uitk-ag-theme.scss

```diff
@@ -0,0 +1,229 @@
+@import "./uitk-ag-theme-mixin";
+
+.uitk-light .ag-theme-uitk-high,
+.uitk-light .ag-theme-uitk-medium,
+.uitk-light .ag-theme-uitk-low,
+.uitk-light .ag-theme-uitk-touch {
+  --uitk-ag-foreground: #2a2c2f;
+  --uitk-ag-foreground-disabled: #161616;
+  --uitk-ag-header-foreground: #61656e;
+  --uitk-ag-background: white;
+  --uitk-ag-row-background-selected: #b7def6;
+  --uitk-ag-row-color-hover: white;
+  --uitk-ag-row-borderColor: #e2e2e2;
+  --uitk-ag-header-column-separator-color: #c5c9d0;
+  --uitk-ag-cell-border-color-editable: #84878e;
+
+  //rgba(36, 37, 38, 0.2);
+  --uitk-ag-row-shadow-hover: 0 2px 5px 0 #24252633, 0 -1px 0 0 #b4b7be;
+  --uitk-ag-row-borderColor-selected: transparent;
+  --uitk-ag-color-focus: #2d81bd;
+
+  --uitk-ag-border-color: #e2e2e2;
+  --uitk-ag-tab-borderColor-active: #e06519;
+  --uitk-ag-checkbox-color: black;
+
+  --uitk-ag-button-color: #161616;
+  --uitk-ag-button-background-hover: #d9dde3;
+
+  --uitk-ag-toggle-background-checked: #2670a9;
+  --uitk-ag-header-icon-color: #61656e;
+  --uitk-ag-menu-header-icon-color: #61656e;
+  --uitk-ag-icon-color: #61656e;
+  --uitk-ag-menu-option-color-hover: #f2f4f6;
+  --uitk-ad-side-buttons-background: #f2f4f6;
+  --uitk-ag-toggle-button-background: #84878e;
+  --uitk-ag-menu-color: #61656e;
+  --uitk-ag-filter-border-color: #eaedef;
+  --uitk-ag-filter-apply-button-color: #242526;
+  --uitk-ag-list-item-color-hover: #b7def6;
+
+  --uitk-ag-column-drop-cell-horizontal-background: #eaedef;
+  --uitk-ag-pagination-button-color: #4c505b;
+  --uitk-ag-pagination-button-color-disabled: rgba(22, 22, 22, 0.4);
+
+  --uitk-ag-placeholder-color: #61656e;
+}
+
+.uitk-dark .ag-theme-uitk-high,
+.uitk-dark .ag-theme-uitk-medium,
+.uitk-dark .ag-theme-uitk-low,
+.uitk-dark .ag-theme-uitk-touch {
+  --uitk-ag-foreground: white;
+  --uitk-ag-foreground-disabled: white;
+  --uitk-ag-header-foreground: white;
+  --uitk-ag-background: #242526;
+  --uitk-ag-row-background-selected: #155c93;
+  --uitk-ag-row-color-hover: #2f3136;
+  --uitk-ag-row-borderColor: #e2e2e2;
+  --uitk-ag-header-column-separator-color: white;
+  --uitk-ag-cell-border-color-editable: #74777f;
+
+  // rgba(36, 37, 38, 0.2)
+  --uitk-ag-row-shadow-hover: 0 2px 5px 0 #24252633, 0 -1px 0 0 #b4b7be;
+  --uitk-ag-row-borderColor-selected: #2d81bd;
+  --uitk-ag-color-focus: #2d81bd;
+
+  --uitk-ag-border-color: #e2e2e2;
+  --uitk-ag-tab-borderColor-active: #e06519;
+  --uitk-ag-checkbox-color: white;
+
+  --uitk-ag-button-color: white;
+  --uitk-ag-button-background-hover: #d9dde3;
+
+  --uitk-ag-toggle-background-checked: #2670a9;
+  --uitk-ag-header-icon-color: white;
+  --uitk-ag-menu-header-icon-color: white;
+  --uitk-ag-icon-color: white;
+  --uitk-ag-menu-option-color-hover: #2f3136;
+  --uitk-ad-side-buttons-background: #2f3136;
+  --uitk-ag-toggle-button-background: #84878e;
+
+  --uitk-ag-menu-color: white;
+  --uitk-ag-filter-border-color: #44484f;
```

Comment:
-borderColor (maybe worth just doing a ctrl F on -border-color and changing them all to -borderColor so none are missed)

Reply:
Done